packages/fx-core/src/plugins/resource/spfx/v3/index.ts

```diff
@@ -0,0 +1,158 @@
+import { hooks } from "@feathersjs/hooks/lib";
+import {
+  AzureSolutionSettings,
+  err,
+  FxError,
+  IConfigurableTab,
+  Inputs,
+  IStaticTab,
+  ok,
+  QTreeNode,
+  Result,
+  TokenProvider,
+  v2,
+  v3,
+  Void,
+} from "@microsoft/teamsfx-api";
+import { Service } from "typedi";
+import { CommonErrorHandlerMW } from "../../../../core/middleware/CommonErrorHandlerMW";
+import { BuiltInFeaturePluginNames } from "../../../solution/fx-solution/v3/constants";
+import { ensureSolutionSettings } from "../../../solution/fx-solution/utils/solutionSettingsHelper";
+import { TabSPFxItem } from "../../../solution/fx-solution/question";
+import { LoadManifestError, SaveManifestError, SPFxAlreadyExistError } from "./error";
+import * as uuid from "uuid";
+import {
+  frameworkQuestion,
+  SPFXQuestionNames,
+  webpartDescriptionQuestion,
+  webpartNameQuestion,
+} from "../utils/questions";
+import { DeepReadonly } from "@microsoft/teamsfx-api/build/v2";
+import { SPFxPluginImpl } from "./plugin";
+import { ManifestTemplate } from "../utils/constants";
+import * as util from "util";
+
+@Service(BuiltInFeaturePluginNames.spfx)
+export class SPFxPlugin implements v3.FeaturePlugin {
```

Comment:
Suggest rename class name to SpfxPluginV3

Reply:
updated